pkg/importer/dataStream.go

```diff
@@ -905,14 +909,18 @@ func (d *DataStream) copy(dest string) error {
 			return err
 		}
 	} else {
-		if util.GetAvailableSpace(d.ScratchDataDir) <= int64(0) {
+		if d.VolumeMode != string(v1.PersistentVolumeBlock) && util.GetAvailableSpace(d.ScratchDataDir) <= int64(0) {
```

Comment:
It is very possible (even likely) that we will require scratch space when we have a block device.

Reply:
correct. Currently we do not (phase 1)

Reply:
The else part of this code, is ONLY used for when we use scratch space, trying to modify it for a streaming case where scratch space is not needed will only make it complicated and unreadable.

Reply:
Not in this feature as I do not support scratch space for block device. This will be changed in phase II.